src/routes/playerRouter.ts

```diff
@@ -28,9 +28,14 @@ playerRouter.get('get-all-players',
     playerController.getAllPlayers
 );
 
+playerRouter.get('get-players-by-team',
```

Comment:
This route should be in the teams router. Even though it is okay staying in the players controller (since it is using the player entity), in the RESP framework, we would put this in the teams route. `${baseRoute}/:id/players`